src/nav/infrastructure.yml

```diff
@@ -618,13 +618,21 @@ pages:
           - title: Microsoft SQL Server integration
             path: /docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql-server-monitoring-integration
           - title: MySQL integration
-            path: /docs/infrastructure/host-integrations/host-integrations-list/mysql-monitoring-integration
+            pages:
+              - title: Installing the MySQL integration
+                path: /docs/infrastructure/host-integrations/host-integrations-list/mySQL/mysql-integration
+              - title: MySQL integration's advanced configuration
```

Comment:
I'd call this one "MySQL advanced config" for the short, left panel title

Reply:
And "Install MySQL"

Reply:
Thanks! 👏🏻